source/style-guide/information-types/reference.txt

```diff
@@ -0,0 +1,96 @@
+.. _meta-reference-prototype:
+
+========================
+Reference Page Prototype
+========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/info-typing/fact-general-guidance.rst
+
+A reference page provides quick information to experienced users of a product. 
+Reference pages may describe the use of a method, operator, or database command. 
+Information presented as a bulleted list or table is usually a reference page. 
```

Comment:
Suggestion:

I think it would be more consistent to describe what the page contains rather than explain that certain formats of information are what make it a reference page.

This prior description on the information-typing.txt seems more accurate since other types such as a task may contain bulleted lists:
> Apart from straightforward reference manual pages, reference pages can include bulleted lists or tables of information.